webapp/channels/src/components/overlay_trigger.tsx

```diff
@@ -6,9 +6,7 @@ import {OverlayTrigger as OriginalOverlayTrigger} from 'react-bootstrap'; // esl
 import type {OverlayTriggerProps} from 'react-bootstrap';
 import {IntlContext} from 'react-intl';
 import type {IntlShape} from 'react-intl';
-import {Provider} from 'react-redux';
-
-import store from 'stores/redux_store';
```

Comment:
Do we want to add (or maybe we already have) some linting limitation to import this? Or some kind of deprecated sign?

Reply:
I was considering that at some point, but I hadn't really decided on it. There are good reasons to access it directly, so I don't want to completely disallow it, but I'd want to discourage directly accessing the store directly